src/compo/viirs_l1bnc2ioda.py

```diff
@@ -158,6 +172,12 @@ def _read(self):
                 orbit_ad[:] = 1
             self.varAttrs['satelliteAscendingFlag', metaDataName]['description'] = '0=descending, 1=ascending'
 
+            # NASA VIIRS stored reflectance need to divide by
+            # cosine of solar zenith angle to get true reflectance
+            sec_term = 1.
+            if self.divide_cos_solarza:
+                sec_term = np.cos(solar_za * deg2rad) ** -1
```

Comment:
I am pretty sure that `1.0/a` operates on computers faster than `a**-1` so I would suggest making it a division not power.  Otherwise, I am OK with the changes.

Reply:
It used to be 1. / np.cos(...), but it triggered a Runtime Warning of invalid value in true_divide (something like that).
Interestingly, I can't reproduce it now (on s4).
I'll bring it back.